core/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -722,27 +715,29 @@ private static boolean awaitTermination(
      * Returns <code>true</code> if the given pool was terminated successfully. If the termination timed out,
      * the service is <code>null</code> this method will return <code>false</code>.
      */
-    public static boolean terminate(ThreadPool pool, long timeout, TimeUnit timeUnit) {
-        if (pool != null) {
+    public static boolean terminate(ThreadPool threadPool, long timeout, TimeUnit timeUnit) {
```

Comment:
Can you revert all changes to this method, they are cosmetic?

Reply:
sure